testing/gnome-control-center/APKBUILD

```diff
@@ -0,0 +1,71 @@
+# Contributor: Rasmus Thomsen <oss@cogitri.dev>
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
+pkgname=gnome-control-center
+pkgver=3.32.1
+pkgrel=1
```

Comment:
```suggestion
pkgrel=0
```

Reply:
Ah right, still am used to Void where 1 is the default. Thanks for the pointer!